store/keystore_test.go

```
@@ -140,7 +134,15 @@ func TestKeystore(t *testing.T) {
 	h.Write(tosigndata)
 	hashed := h.Sum(nil)
 
-	err = rsa.VerifyPKCS1v15(&k.private.PublicKey, crypto.SHA256, hashed, s)
+	//generate pubkey for golang stdlib
+	block, _ := pem.Decode([]byte(expectedaspem))
```

Why explicitly ignore the error here?

---

I copied that from the old code in store/keystore.go loadFromPem() ... will fix it.

---

Thanks :100:

---

thats actually not the error. its the undecoded rest.

https://golang.org/pkg/encoding/pem/#Decode

i will leave it as it is.